arviz/tests/external_tests/test_data_pystan.py

```diff
@@ -1,17 +1,31 @@
 # pylint: disable=no-member, invalid-name, redefined-outer-name
+import importlib
 from collections import OrderedDict
+
 import numpy as np
 import pytest
 
 from arviz import from_pystan
+
 from ...data.io_pystan import get_draws, get_draws_stan3  # pylint: disable=unused-import
 from ..helpers import (  # pylint: disable=unused-import
     chains,
     check_multiple_attrs,
     draws,
     eight_schools_params,
+    importorskip,
     load_cached_models,
     pystan_version,
+    running_on_ci,
+)
+
+# Check if either pystan or pystan3 is installed
+pystan_installed = (importlib.util.find_spec("pystan") is not None) or (
+    importlib.util.find_spec("stan") is not None
+)
+pytestmark = pytest.mark.skipif(
+    not pystan_installed & (not running_on_ci()),
```

Comment:
not pystan_installed & (not running_on_ci())

not True & (not True) == True

(not True) & (not True) == False

Reply:
Or you can use

    not ( pystan_installed |  running_on_ci() )

Reply:
I think that logic is cleaner. But shouldn't the current logic work because it equates to False, so it shouldn't get skipped

Reply:
It is currently true (missing parenthesis)

    not True & (not True) == True

Reply:
Ah yes okay. I updated that, will see if it does the trick. And I see that there are some built-in emcee skips looking for emcee3, so that should still be okay